build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("org.springframework.boot") version "3.1.3"
+    id("io.spring.dependency-management") version "1.1.3"
+    id("io.gitlab.arturbosch.detekt") version("1.23.1")
+    kotlin("jvm") version "1.9.0"
+    kotlin("plugin.spring") version "1.9.0"
+    kotlin("plugin.jpa") version "1.9.0"
+    kotlin("plugin.allopen") version "1.9.0"
+}
+
+repositories {
+//    mavenLocal()
+//    maven {
+//        url = uri("https://repo.maven.apache.org/maven2/")
+//    }
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-validation")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    runtimeOnly("com.mysql:mysql-connector-j")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+}
+
+//dependencies {
+//    api("org.springframework.boot:spring-boot-starter-data-jpa:3.1.3")
+//    api("org.springframework.boot:spring-boot-starter-web:3.1.3")
+//    api("com.fasterxml.jackson.module:jackson-module-kotlin:2.15.2")
+//    api("org.jetbrains.kotlin:kotlin-reflect:1.8.22")
+//    api("org.jetbrains.kotlin:kotlin-stdlib:1.8.22")
+//    runtimeOnly("com.mysql:mysql-connector-j:8.0.33")
+//    testImplementation("org.springframework.boot:spring-boot-starter-test:3.1.3")
+//}
+
+group = "com.ajaxproject"
+version = "0.0.1-SNAPSHOT"
+description = "warehouse"
+
+//publishing {
+//    publications.create<MavenPublication>("maven") {
+//        from(components["java"])
+//    }
+//}
```

Comment:
please, clean up your script from commented code:
- uncomment
- or remove at all

TODO is the exception case :)